frontend/wallet/src/render/components/TransactionCell.re

```diff
@@ -318,22 +287,28 @@ module InfoSection = {
        | StakingReward(rewards, _) =>
          <>
            <MainRow>
-             <Link> {React.string(expanded ? "Collapse" : "Details")} </Link>
+             <Link>
+               {React.string(expanded ? "Collapse" : "Show fee transfers")}
```

Comment:
nit: maybe we should use some other copy in the UI - "fee transfers" is not really clear.  Maybe "staking fees earned"

Reply:
Yeah but fees can be for snarking, staking, and snarking the coinbase. I think fee transfers makes some sense since it reflects what's actually going on, but down to get different copy.

Reply:
fair, we can hammer it out later